### ISSUE TITLE
Error when using babel-preset-minify configured to mangle toplevel names

### ISSUE THREAD

**Issue by BenoitZugmeyer** [rollup/rollup-plugin-babel#268]
I have a similar issue as #267  when using `babel-preset-minify` with the option `{ mangle: { topLevel: true } }`.

I made a minimal repo to reproduce:

```
git clone https://gist.github.com/ae41d5ee0e79100e65c1dee82287490e preflight-error
cd preflight-error
npm i
node_modules/.bin/rollup -c
```
This outputs an error like this:
```
main.js → bundle.js...
[!] (babel plugin) Error: An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!
main.js
Error: An unexpected situation arose. Please raise an issue at https://github.com/rollup/rollup-plugin-babel/issues. Thanks!
    at error (/.../node_modules/rollup/dist/rollup.js:3460:30)
    at Object.error (/.../node_modules/rollup/dist/rollup.js:21136:17)
    at /.../node_modules/rollup-plugin-babel/dist/rollup-plugin-babel.cjs.js:72:9
    at Object.transform$1 (/.../node_modules/rollup-plugin-babel/dist/rollup-plugin-babel.cjs.js:174:18)
    at /.../node_modules/rollup/dist/rollup.js:20847:25
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:752:3)
```

In the preflight transformed code, the `function _inherits` gets mangled, so the plugin can't find it.

As a temporary fix, I use the `exclude` option to make sure `_inherit` isn't mangled.

```
      rollupPluginBabel({
        presets: [
          [
            "minify",
            {
              mangle: {
                topLevel: true,
                exclude: { _inherits: true },
              },
            },
          ],
        ],
      }),
```

**Comment by Andarist** [comment]
Thanks for the report, I'm not sure what can we do about it though. The whole preflightCheck is shady and I'd like to remove it in next major version.

Probably switching to https://github.com/Comandeer/rollup-plugin-babel-minify for minifying purposes would fix your issue.

**Comment by christophehurpeau** [comment]
I have the same issue, but I only use the plugin `babel-plugin-minify-dead-code-elimination` to perform some optimizations.
Would it be possible to add an option to skip the preflight check before the next major ?

**Comment by Andarist** [comment]
Has been fixed by https://github.com/rollup/rollup-plugin-babel/commit/d3462bacdd6b853e08162346b77a2cfd8413160f - this will be a part of upcoming v5 release. 

> Would it be possible to add an option to skip the preflight check before the next major ?

Yes, it's already implemented and released in 5.0.0-alpha.1
